src/Filters/atoms/DateTimeBlock.tsx

```diff
@@ -168,20 +169,14 @@ const DateTimeBlock : React.FC<IProps> = ({
     }
   },[date, allowAfterMidnight]);
 
-  const onBlurInputs = useCallback(() =>{
-    if(displayHours === '0' || displayHours === ''){
-      setDisplayHours('00');
+  const onBlurInputs = useCallback((blurValue: string, value:string) =>{
+    const convertedValue = getFormattedTime(blurValue);
+    if(value === 'Minutes'){
```

Comment:
Having a string like this is fine but it would be better to keep it as lower case. As the options here are limited we might be able to get aways with `mins` or even just `m` so minutes.

Reply:
I have changed.
Minutes --> mins
Hour --> hours